AutomatedTesting/Gem/PythonTests/scripting/scripting_utils/script_canvas_component.py

```diff
@@ -36,11 +36,7 @@ class VariableState(Enum):
 
 class ScriptCanvasComponent:
```

Comment:
Let's relocate this file to the `editor component behaviors` in _AutomatedTesting/Gem/PythonTests/EditorPythonTestTools/editor_python_test_tools/editor_component/_

Additionally, we may want to refactor to follow the get/set properties then add the custom validators.